main.py

```diff
@@ -28,7 +27,14 @@
 profanity.load_censor_words_from_file('banlist.txt')
 available_songs = get_all_music_available()
 cache = LRUCache()
-mongo_client = MongoClient('mongodb://localhost/')
+
+if os.environ.get("ACE_MONGODB"):
```

Comment:
Hmmm, there is `settings.py` already, maybe it can be used here...

Reply:
You're probably right. Consistence is key. You should move it to setting.py and check for None.

Sorry for the trouble

Reply:
Should it be gitignored, btw?

Reply:
I'm not really sure. Git ignoring it makes it more of a hassle to introduce changes, like this one for instance.

Also I was thinking maybe env variables is better, after all in some PaaS platforms is the only thing you can edit.

So better leave it as it is like env variable. I'll take a look at it when I'm on my PC.
Thanks